woocommerce/woocommerce
# Fix Marketplace connect URL for multi-sites

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR fixes the issue related to market-place settings not being included on the in-app extensions page when site is running on sub-directory based multi-site setup.

Closes # .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create a multi-site on https://jurassic.ninja/ with latest WooCommerce release.
2. Create a site in network admin `/wp-admin/network/sites.php`
3. Visit the in-app extension page of the newly created site `/<site_path>/wp-admin/admin.php?page=wc-admin&tab=my-subscriptions&path=%2Fextensions`
4. Notice that WCCOM connect button isn't working. The URL is empty on the button link.
5. Checkout this branch
6. Run `npm run build:zip` from `<woocommerce repo root>/plugins/woocommerce`
7.  Upload the build `<woocommerce repo root>/plugins/woocommerce/woocommerce.zip` to the above multi-site
8.  Visit the in-app extension page of the newly created site `/<site_path>/wp-admin/admin.php?page=wc-admin&tab=my-subscriptions&path=%2Fextensions`
9. Note that the WCCOM connect button is functioning as expected.



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

Fix: Unable to connect sub-directory based multi-sites to WooCommerce.com.

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
